src/ui/components/searchResult/SearchResult.tsx

```diff
@@ -79,6 +80,9 @@ export const VuiSearchResult = forwardRef<HTMLDivElement | null, Props>(
             {date && <VuiTextColor color="subdued">{date} &#8212; </VuiTextColor>}
             {pre} <strong>{text}</strong> {post}
           </p>
+          <VuiLink href={recommendedUrl(text)} target="_blank">
+            <i><h5>+ Explore this concept more</h5></i>
```

Comment:
Let's add a config option to include (or not) this text below search results.

Reply:
And let's add something about recommendation in the README - how to configure it? how does it work? (with a link to the docs about our semantics argument in search query API)